src/Symfony/Component/Notifier/Bridge/Esendex/EsendexTransport.php

```diff
@@ -44,7 +46,7 @@ public function __construct(string $token, string $accountReference, string $fro
 
     public function __toString(): string
     {
-        return sprintf('esendex://%s', $this->getEndpoint());
+        return sprintf('esendex://%s?accountreference=%s&from=%s', $this->getEndpoint(), $this->accountReference, $this->from);
```

Comment:
I also updated the `__toString()` method to display accountreference and from, like in the other bridges